src/classes/REL_Relationships_TEST.cls

```diff
@@ -859,5 +859,85 @@ private class REL_Relationships_TEST {
             REL_RelationshipsViewer_CTRL.RelationshipInfo ri = REL_RelationshipsViewer_CTRL.getRelationshipInfo();
             system.assertNotEquals(null, ri);                       
         }
-    }   
+    }
+
+    public static testMethod void testDeletingMutuallyReferentialRelationshipsDoesNotPreventDeleteOfReciprocals() {
```

Comment:
Thanks, Lurch.  I agree.  I'm rejecting this pull request until this issue has been remedied.